manifests/2.2.2/opensearch-2.2.2.yml

```diff
@@ -22,6 +22,15 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version
+  - name: ml-commons
+    repository: https://github.com/opensearch-project/ml-commons.git
+    ref: '2.2'
+    platforms:
+      - linux
+      - windows
```

Comment:
I am not sure if windows should be added here. Even Common utils does not have it. Maybe remove it for now?

Reply:
Yeah windows is not supported for 2.2.x. please remove it. Thanks!